android/run-local-test/local.json

```
@@ -0,0 +1,22 @@
+{
+    "user": "BROWSERSTACK_USERNAME",
+    "key": "BROWSERSTACK_ACCESS_KEY",
+    "capabilities": {
+        "app" : "bs://<app-id>",
+        "bstack:options": {
+            "projectName": "Pytest Browserstack",
+            "buildName": "browserstack-build-android",
```

```suggestion
            "buildName": "browserstack-build-1",
```

we use this specific buildName for QIG and data tracking